packages/mobile/src/paymentRequest/PaymentRequestReviewCard.tsx

```diff
@@ -0,0 +1,122 @@
+import HorizontalLine from '@celo/react-components/components/HorizontalLine'
```

Comment:
This file is new, it was broken out of TransferReviewCard so we can simplify it and more easily customize the UI for sending a payment request

Reply:
Nice!